.github/workflows/docker.yml

```diff
@@ -5,8 +5,8 @@ on:
   release:
     types:
     - created
-  schedule:
-    - cron:  '0 0 * * *'
+  #  schedule:
```

Comment:
Why disable?

Reply:
I don't know why we want to build this nightly. It is pinned to a Geodesic version and we do not update Geodesic that often. It just seems a waste of resources to build it nightly.